Cargo.toml

```diff
@@ -1,23 +1,37 @@
 [workspace]
-members = ["ceramic-kubo-rpc", "ceramic-one"]
+members = [
+    "core",
+    "event",
```

Comment:
I like these shorter names without the `ceramic-` stutter I'll likely rename all the directories in the repo to match this pattern.

Reply:
Yeah, I like the crate names with the full `ceramic-` indicator, but since we're in a ceramic project, tend to drop the prefix.